src/backend/Modules/Account/Timebox.Account.Application.Interfaces/Services/IPasswordService.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System.Threading.Tasks;
+
+namespace Timebox.Account.Application.Interfaces.Services
+{
+    public interface IPasswordService
+    {
+        bool IsStrongPassword(string password);
```

Comment:
Like this, what rules have we gone for ?

Reply:
I will add a comment.